Python/Product/Analysis/Parsing/Parser.cs

```diff
@@ -864,6 +954,15 @@ struct Name {
         private Statement ParseExprStmt() {
             Expression ret = ParseTestListAsExpr();
 
+            if (PeekToken(TokenKind.Colon) && _langVersion >= PythonLanguageVersion.V36) {
+                ret = ParseNameAnnotation(ret);
+                if (!PeekToken(TokenKind.Assign)) {
```

Comment:
The else should be handled here as well, otherwise you can do "foo: int = 2 = 3" which doesn't appear to be legal